### PR TITLE
doc: add Graphic configuration explanation

### PR DESCRIPTION
**Pull request by cboesch-gpsw**

### CODE REVIEW

File: doc/expl/graphicconfig.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+Color and alpha blending, depth, stencil and culling are controlled by the
+`GraphicConfig` node.
+
+`node.gl` is aligned with the traditional blending mode concepts from graphics
+APIs. For example, color and alpha blending involves source and destination
+factors for color channels or alpha, and a blending operation (addition,
+subtraction, etc). This document will assume you are familiar with these
+concepts.
+
+For color and alpha blending, you will interested in particular in the
```
Comment (sfranz-gpsw) [inline doc/expl/graphicconfig.md:12]: you will interested  in particular in -> you will be particularly interested in

**Review by sfranz-gpsw — APPROVED** [review]
Thank you for the doc! Except for my little comment, LGTM

File: doc/expl/graphicconfig.md (hunk 1 above)
Comment (cboesch-gpsw) [reply doc/expl/graphicconfig.md]: thanks, fixed

**Comment by mbouron-gpsw** [comment]
LGTM.

Not directly related to this PR but we use "graphic" in a lot of place of the codebase but it looks like the right term is "graphic*s*". We might want to fix that at some point.

**Review by mbouron-gpsw — APPROVED** [review]

**Comment by ubitux** [comment]
> LGTM.
>

Thanks, will merge
 
> Not directly related to this PR but we use "graphic" in a lot of place of the codebase but it looks like the right term is "graphic_s_". We might want to fix that at some point.

The document use "graphic" in the following situations:
1. to quote `GraphicConfig` (which I can't change)
2. in the title as "Graphic configuration" but I'd like to keep it this way for now since it matches `GraphicConfig`
3. for "Graphic state" -- somehow it sounds more correct to me that way but I might be wrong

I reference OTOH "Graphics API"...

So yeah, it's unclear to me what's the correct way to write it, but we should probably change the node name first if it's incorrect.

**Pull request merged**